client/src/app/core/repositories/users/user-repository.service.ts

```diff
@@ -520,4 +613,32 @@ export class UserRepositoryService extends BaseRepositoryWithActiveMeeting<ViewU
             throw new PreventedInDemo();
         }
     }
+
+    private toDecimal(number?: number): string {
```

Comment:
Please move it out of this repo and into an own file (as a stand-alone function). It will be used by polls later, too.